openslides/global_settings.py

```diff
@@ -11,7 +11,7 @@
 
 AUTHENTICATION_BACKENDS = ('openslides.users.auth.CustomizedModelBackend',)
 
-LOGIN_URL = '/login/'
+LOGIN_URL = '/users/'
```

Comment:
Then we need an issue and TODO to change this to '/' when the root is ready via AngularJS.